### PR TITLE
Structured log migration for pkg/util/netsh

### PR DESCRIPTION
**Pull request by jialaijun**
in /pkg/util/netsh/netsh.go

**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it:**

Ref:

- [keps/sig-instrumentation/1602-structured-logging](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging)

- [Structured Logging migration instructions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md)

**Does this PR introduce a user-facing change?:**

`Migrate some log messages to structured logging `

### CODE REVIEW

File: pkg/util/netsh/netsh.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ func New(exec utilexec.Interface) Interface {
 
 // EnsurePortProxyRule checks if the specified redirect exists, if not creates it.
 func (runner *runner) EnsurePortProxyRule(args []string) (bool, error) {
-	klog.V(4).Infof("running netsh interface portproxy add v4tov4 %v", args)
+	klog.V(4).InfoS("running netsh interface portproxy add v4tov4", "args", args)
```
Comment (shubheksha) [inline pkg/util/netsh/netsh.go:69]:
```suggestion
	klog.V(4).InfoS("Running netsh interface portproxy add v4tov4", "args", args)
```

File: pkg/util/netsh/netsh.go [hunk 2]
```diff
@@ -85,7 +85,7 @@ func (runner *runner) EnsurePortProxyRule(args []string) (bool, error) {
 
 // DeletePortProxyRule deletes the specified portproxy rule.  If the rule did not exist, return error.
 func (runner *runner) DeletePortProxyRule(args []string) error {
-	klog.V(4).Infof("running netsh interface portproxy delete v4tov4 %v", args)
+	klog.V(4).InfoS("running netsh interface portproxy delete v4tov4", "args", args)
```
Comment (shubheksha) [inline pkg/util/netsh/netsh.go:88]:
```suggestion
	klog.V(4).InfoS("Running netsh interface portproxy delete v4tov4", "args", args)
```

File: pkg/util/netsh/netsh.go [hunk 3]
```diff
@@ -114,20 +114,20 @@ func (runner *runner) EnsureIPAddress(args []string, ip net.IP) (bool, error) {
 
 	exists, _ := checkIPExists(ipToCheck, argsShowAddress, runner)
 	if exists == true {
-		klog.V(4).Infof("not adding IP address %q as it already exists", ipToCheck)
+		klog.V(4).InfoS("not adding IP address, as it already exists", "ipToCheck", ipToCheck)
```
Comment (shubheksha) [inline pkg/util/netsh/netsh.go:117]:
```suggestion
		klog.V(4).InfoS("Not adding IP address, as it already exists", "IP", ipToCheck)
```

File: pkg/util/netsh/netsh.go [hunk 4]
```diff
@@ -114,20 +114,20 @@ func (runner *runner) EnsureIPAddress(args []string, ip net.IP) (bool, error) {
...
 		return true, nil
 	}
 
 	// IP Address is not already added, add it now
-	klog.V(4).Infof("running netsh interface ipv4 add address %v", args)
+	klog.V(4).InfoS("running netsh interface ipv4 add address", "address", args)
```
Comment (shubheksha) [inline pkg/util/netsh/netsh.go:122]:
```suggestion
	klog.V(4).InfoS("Running netsh interface ipv4 add address", "args", args)
```

**Review by shubheksha — CHANGES_REQUESTED** [review]

**Review by caseydavenport — APPROVED** [review]
/lgtm

**Comment by pacoxu** [comment]
/hold
as there are still some comments.

/release-note-none

**Comment by dims** [comment]
@pacoxu is this ready?

**Comment by pacoxu** [comment]
> @pacoxu is this ready?

`running -> Running` Only waiting for this. Not sure if it can be a follow up as the author is not active for about 4 months.

**Comment by pacoxu** [comment]
/hold cancel
/good-first-issue 
to do a cleanup as the author has no response.

**Comment by calvin0327** [comment]
I would like to work in this and submitted a PR as a supplement.

**Pull request merged**